registry.json

```diff
@@ -2880,5 +2880,35 @@
       "source": "https://github.com/hashgraph",
       "documentation": "https://docs.hedera.com"
     }
+  },
+  {
+    "id": "nativeinjective",
+    "name": "NativeInjective",
+    "displayName": "Native Injective",
+    "coinId": 10000060,
```

Comment:
We may need to define some rules for this kind of coin id

Reply:
I came up with `10000060` because it uses 60 as cointype in derivation path and it doesn't have any numeric chainId or slip44 id for native token.

Reply:
@catenocrypt @Milerius  maybe we should follow how we add Osmosis?

Reply:
Osmosis has it's own coinID. For most Eth clones, we derived internal coinID from chainID. But this does not have chainID, as it's not an Eth clone, yet it uses Eth derviation path. Nor does it have a coinID.
For me it's OK to have `10000060`, as it uses 60 in derivation path, but I'm open to other suggestions.

Reply:
Or we can just plus 1 from some base number if no slip44 or chain id, this seems the easiest way